### DISCUSSION TITLE
What nice are you in? 🎉

### DISCUSSION THREAD

**Discussion by pH-7** [pH7Software/pH7-Social-Dating-CMS#817] [General]
Following to **Best Niche Ideas for your Online Dating Web App** 👉 https://ph7cms.com/niche-ideas-for-dating-web-app I'm curious to know if you build a niche website or generic one 🤔 Are you more in a "Social Networking for Dogs" approach, Community for Lawers, or on something different...?

**Comment by Undefined-Variables** [comment]
I have written my own custom template and had to take a step back and disable a lot of the default features because there was just to much going on and not in a good way. All the features work and in the right situation many of them would be useful but for what I needed it was too much! Think having the user dashboard and wall just makes it over confusing... 

@pH-7 you’re more than welcome here to have a look (I will email you the URL)! I’m just not publicly sharing it! Do reply and give me your thoughts!

**Reply by Undefined-Variables** [reply]
What is the point of a discussion board that is not even being used by the lead dev? @pH-7 is me directly talking to you!

**Comment by polynamaude** [comment]
@Undefined-Variables I doubt he's gonna have time to answer this question.

**Comment by polynamaude** [comment]
@Undefined-Variables But if you say that you can't use a east european character then you'll get a fix in 24 hours. Akward !

**Comment by polynamaude** [comment]
@Undefined-Variables And the problem here is not his lack of time but that he doesn't open his project to contributor. Most of the code that got suggested never get included. If only the reason was that this software is greatly coded without bug then I'll understsand he doesn't want to have problem...
But there's more than enough bugs that it would be worth to write from scratch.
Maybe he doesn't feel like it's worth taking the risk someone else get some credit for doing a nice job....

**Comment by Undefined-Variables** [comment]
@polynamaude I agree and even I am loosing interest due to the lack of community spirit and input to this project. There are open issues that have not been solved and repeated instances of identical issues that should have been solved but never have been! 

I asked for advice on the mutual friend error months ago and if he can’t take the time to answer my question when I spend time answering questions on his behalf then I won’t bother answering them. This is a chain of command and it doesn’t work with a broken link.

**Reply by polynamaude** [reply]
@Undefined-Variables But strangely enough, he just answered a question relating to East European language...
See the screenshot I sent you by email...
https://github.com/pH7Software/pH7-Social-Dating-CMS/issues/825#issuecomment-716151283
I stole a tag line...

**_Use fricking code tags!!!!
I want x => I would like x, I need help => I would like help, Need fast => Go and pay someone to do the job..._**

**Comment by polynamaude** [comment]
@Undefined-Variables People just take take take take take and don't give back.
They don't even take a minute to say thanks and they close issue. And if you tell them to be a bit nicer, that you ain't micro$oft tech support, to start helping themselves... what happen ? you get a message from PH that you are supposed to be a superhero and always be nice with people... yeah, ignore them !

One of the big problem is that the publicity made about "becoming rich without efforts and knowledge" just drive some dumb people to the git page. They don't know much about computing but think they have "the project of their lifetime". They forget that if it would be so simple then other people would have done it before them !

**Reply by Undefined-Variables** [reply]
Yeah exactly.

**Comment by polynamaude** [comment]
@Undefined-Variables Now you understand why I responded the way I did...
It's not something about being rude.

BtW... go buy a ESP2866 board for Arduino. You can get one for 10$. Afterward you can make a easy WiFi DeAuth killer box.
